storybook/src/components/Grid/Grid.stories.tsx

```diff
@@ -66,7 +69,7 @@ type CellStory = StoryObj<typeof cellMeta>
 const StoryTemplate: Story = {
   decorators: [
     (Story) => (
-      <Screen>
+      <Screen className="ams-storybook-show-grid">
```

Comment:
I’d prefer to apply the class name to `<Story />` – but that doesn’t get applied.

Making it an `arg` in `meta` applies it to Grid Cells as well, resulting in an incorrect background pattern.